tests/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
 RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
     ln -s /cache/images /build/images && \
+    ln -s /cache/images /build/virt-builder && \
```

Comment:
> This will throw virt-builder and cockpit images into the same directory. They won't collide on names, but does it really make sense to replicate and distribute them across all of our image stores? Keeping that image around locally seems sufficient.

They are not replicated, unless someone asks for them. They are placed in a cache directory so other containers sharing that cache directory can use the images.

> Are we sure that the virt-builder images never change? I. e. Debian 8 or Ubuntu 16.04 regularly get point releases, and the virt-builder images might get refreshed under the same name. 

The file names are incremented with a number here: http://libguestfs.org/download/builder/index.asc  virt-builder itself caches these images and would fail if it didn't change the names. 

> TBH I'd treat them as an implementation detail of virt-builder and not expose them to our mirroring infra.

They are part of our cache, not part of any mirroring.